pkgs/top-level/all-packages.nix

```diff
@@ -21110,6 +21110,8 @@ with pkgs;
 
   tup = callPackage ../development/tools/build-managers/tup { };
 
+  truffleHog = python3.pkgs.callPackage ../tools/security/truffleHog { };
```

Comment:
Please pass `python3` or `python3Packages` as an argument instead. This will allow overriding of specific dependencies' versions if necessary.

Reply:
Like this?
`  truffleHog = pythonPackages.callPackage ../tools/security/truffleHog { };`